.github/workflows/test.yml

```
@@ -11,7 +11,25 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
-
+    # Docker Hub image that `container-job` executes in
+    container: node:18
+    services:
+      # Label used to access the service container
+      db:
+        # Docker Hub image
+        image: postgres:14.8
+        ports: ['5432:5432']
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: Admin@1234
+          POSTGRES_USER: postgres
+          POSTGRES_DB: web_scrapers
```

The datatabse name should be named following the pattern `project_name_environment`

```suggestion
          POSTGRES_DB: ic_nextjs_test
```